### ISSUE TITLE
Allow object creation of any level

### ISSUE THREAD

**Issue by webberwang** [codelab-ai/codelab.ai#167]
<img width="519" alt="localhost_4200" src="https://user-images.githubusercontent.com/26264925/95720581-4cb2aa00-0c26-11eb-9409-872cdb4486b9.png">

should give us

```
{
  "user": {
    "address": {
      "number": 1200,
      "street": "Park ave.",
      "city": "London"
    }
  }
}
```

Use this package https://www.npmjs.com/package/dot-object

Add a spec for this as well

**Comment by webberwang** [comment]
<img width="316" alt="localhost_4200" src="https://user-images.githubusercontent.com/26264925/95972986-faa98a00-0dc7-11eb-9279-f7156eeb7960.png">

Also strip away key value to make array of objects. we'll want to move this logic into custom form component

**Comment by webberwang** [comment]
Closed by #223

**Comment by VienNguyen24hdev** [comment]
via Hubstaff
User: Nguyen Luong Vien

Project: Codelab - https://app.hubstaff.com/projects/1093588
Date Range: 10/14/20 - 10/25/20
Work session total: 18:09:46
Billable: Yes

Grand total: 18:09:46
